TF_Service_dotNet/TouchFreeTests/VirtualScreenTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System.Numerics;
+using NUnit.Framework;
+using TouchFreeTests.TestImplementations;
+using Ultraleap.TouchFree.Library;
+using Ultraleap.TouchFree.Library.Configuration;
+
+namespace TouchFreeTests
+{
+    public class VirtualScreenTests
+    {
+        [Test]
+        public void Constructor_ValidInputs_ReturnsInstance()
+        {
+            //Given
+            IConfigManager configManager = new TestConfigManager();
+
+            //When
+            VirtualScreen virtualScreen = new VirtualScreen(1080, 1920, 0.4f, 0, configManager);
+
+            //Then
+            Assert.AreEqual(1080, virtualScreen.Width_VirtualPx);
+            Assert.AreEqual(1920, virtualScreen.Height_VirtualPx);
+            Assert.AreEqual(0.4f, virtualScreen.Height_PhysicalMeters);
+            Assert.AreEqual(0.225f, virtualScreen.Width_PhysicalMeters, 0.001);
+            Assert.AreEqual(0, virtualScreen.AngleOfPhysicalScreen_Degrees);
+            Assert.AreEqual(0, virtualScreen.PhysicalScreenPlane.Normal.X);
+            Assert.AreEqual(0, virtualScreen.PhysicalScreenPlane.Normal.Y);
+            Assert.AreEqual(1, virtualScreen.PhysicalScreenPlane.Normal.Z);
+        }
+
+        private int ScreenWidthInPixels = 1080;
+        private int ScreenHeightInPixels = 1920;
+        private float ScreenHeightInMeters = 0.4f;
+
+        private VirtualScreen CreateVirtualScreen(float angleInDegrees)
+        {
+            IConfigManager configManager = new TestConfigManager();
+            return new VirtualScreen(ScreenWidthInPixels, ScreenHeightInPixels, ScreenHeightInMeters, angleInDegrees, configManager);
+        }
+
+        [TestCase(480, 960, 0.1f, 0.2f)]
+        [TestCase(0, 540, 0f, 0.1125f)]
```

Comment:
I didn't know we could put together test cases like this, really useful to know!